Automattic/jetpack
# Openverse: Update icon to reflect the right colors

#### Changes proposed in this Pull Request:

* Updated Openverse SVG icon

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Go to the editor and insert an image or gallery block.
* Click "Select media"
* Double-check that the Openverse icon is displayed with a yellow background and dark spots.

#### Screenshots

Before | After
--- | ---
 <img width="265" alt="Screen Shot 2022-03-24 at 3 39 22 PM" src="https://user-images.githubusercontent.com/1398304/160022040-965567ef-fb26-4e78-bf75-303f6c8f71b8.png"> | <img width="274" alt="Screen Shot 2022-03-24 at 3 38 18 PM" src="https://user-images.githubusercontent.com/1398304/160022059-3d60881b-1b86-4eb1-b2d2-000e1edd46a2.png">
